docker/templates/new-wagtail-app/{{ cookiecutter.app_name }}/README.md

```diff
@@ -0,0 +1,140 @@
+# {{ cookiecutter.app_verbose_name }}
+
+## Cookiecutter Notes
+
+### Preparing for Docker
+
+Before building the Docker container, initialize the database:
+
+```bash
+docker-compose run --rm app python manage.py makemigrations
+```
+
+```bash
+docker-compose run --rm app python manage.py migrate
+```
```

Comment:
Bumping.

Reply:
I think I was under the impression that since this would be a new app, we needed to generate a new initial migrations each time.  But we don't and it makes much more sense to include that file in the template.